src/Razor/test/VSCode.FunctionalTest/tests/CodeLens.test.ts

```diff
@@ -37,7 +37,11 @@ suite('CodeLens', () => {
     });
 
     test('Can resolve CodeLens in .razor file', async () => {
```

Comment:
```suggestion
    test('Can resolve CodeLens in .razor file', async function() {
```

Reply:
Curious, why the difference? To not capture "this"?

Reply:
Yeah I'm not sure exactly how it works under the covers.